docs/gettingstarted/structuring.md

```diff
@@ -1,54 +1,79 @@
 Structuring Your Mod
 ====================
 
-We'll look at how to organize your mod into different files and what those files should do.
+In this section, we'll look at the general convention on how to organize your mod into different packages, and what those packages should do.
 
 Packaging
 ---------
 
-Pick a unique package name. If you own a URL associated with your project, you can use it as your top level package. For example if you own "example.com", you may use `com.example` as your top level package.
+Your mod should have a unique package name to avoid conflicts with anything else that might be loaded in from a Classloader. If you own a domain you would like to associate or have a domain associated with other projects already, you can use it as your **top level package**. For example, if you own the domain *"examplemod.com"*, you can choose to use `com.examplemod` as your top level package.
 
-!!! important
+**HOWEVER**
 
-    If you do not own a domain, do not use it for your top level package. It is perfectly acceptable to start your package with anything, such as your name/nickname, or the name of the mod.
+If you do not own a domain, **do *NOT* use it for your top level package!**  
```

Comment:
The second preferred option is to use your email, so people have a way of associating you with something you own.
EXA: com.gmail.lexmanos